apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/utils.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * External Dependencies
+ */
+import { isDesktop } from '@automattic/viewport';
+/**
+ * Internal dependencies
+ */
+import type { TourAsset } from './types';
+
+/**
+ * This function returns the src string for the current Welcome Tour View based on the current viewport
+ *
+ * @param asset {TourAsset} list of assets for current Welcome Tour View
+ * @returns {string} asset src string
+ */
+export const getImageSrcForView = ( asset: TourAsset ): string => {
+	return ! isDesktop() && asset.mobile ? asset.mobile.src : asset.desktop.src;
+};
```

Comment:
Let's rethink this part and consider using [@automattic/viewport-react](https://www.npmjs.com/package/@automattic/viewport-react) instead. Might mean a component that finds the correct image based on viewport changes. 

Related thing happening in #58009

CC @escapemanuele @razvanpapadopol